multisrc/overrides/madara/mmscans/src/MMScans.kt

```diff
@@ -12,8 +18,76 @@ class MMScans : Madara("MMScans", "https://mm-scans.org", "en") {
 
     override val popularMangaUrlSelector = "div.item-summary a"
     override fun chapterListSelector() = "li.chapter-li"
-    override fun searchMangaSelector() = "a"
+    override fun searchMangaSelector() = ".search-wrap >.tab-content-wrap > a"
+    override fun searchMangaNextPageSelector(): String? = "div.no-posts"
 
+    override fun popularMangaParse(response: Response): MangasPage {
+        runCatching { fetchGenres() }
+        val document = response.asJsoup()
+
+        val mangas = document.select(popularMangaSelector()).map { element ->
+            popularMangaFromElement(element)
+        }
+
+        val hasNextPage = popularMangaNextPageSelector()?.let { selector ->
+            document.select(selector).first()
+        } != null
+
+        return MangasPage(mangas, !hasNextPage)
+    }
+    override fun latestUpdatesParse(response: Response): MangasPage {
+        val document = response.asJsoup()
+
+        val mangas = document.select(latestUpdatesSelector()).map { element ->
+            latestUpdatesFromElement(element)
+        }
+
+        val hasNextPage = latestUpdatesNextPageSelector()?.let { selector ->
+            document.select(selector).first()
+        } != null
+
+        return MangasPage(mangas, !hasNextPage)
+    }
+
+    fun madara_load_more(page: Int, meta_key: String): Request {
```

Comment:
Please use camelCase.

```suggestion
    fun oldLoadMoreRequest(page: Int, metaKey: String): Request {
```

I also suggested a new name for the function as Madara used to do this call before, but was removed for security reasons.

Reply:
Also need to rename the usages of it.